frontend/src/requests/transaction.rs

```diff
@@ -1,19 +1,58 @@
+use serde_json::{json, Value};
+
 use crate::config::Config;
 
 use super::request;
 
-pub async fn ping(config: &Config) -> Result<String, String> {
+pub enum TransactionType {
+    Ping,
+    Pong,
+    PongAll,
+}
+
+pub async fn ping(config: &Config) -> Result<Value, Value> {
     let transaction_url = &config.transaction_url;
     let dest = &config.dest;
-    let full_endpoint = format!("http://{dest}{transaction_url}");
+    let endpoint = format!("http://{dest}{transaction_url}/ping");
+
+    let body = json!({
```

Comment:
place this into an outside form. still not the ideal solution, but good enough until wallet connect

Reply:
This is the next feature that we will implement